tuning/autotune.py

```diff
@@ -707,7 +732,7 @@ def autotune() -> None:
     )
     print(f"Unet candidates compiled in {base_dir}\n")
 
-    print("Bnechmarking unet candidates...")
+    print("Benchmarking unet candidates...")
```

Comment:
nice catch!

Reply:
#27 also applied some changes on this string, not sure should we fix the typo now or later